package.json

```diff
@@ -96,6 +101,7 @@
   },
   "dependencies": {
     "@babel/runtime-corejs2": "^7.0.0",
+    "@modus/more": "0.0.2",
```

Comment:
For me, npm i fails here:

```
npm ERR! 404 Not Found: @modus/more@0.0.2
```

Reply:
@emimuresan can you try manually with 
```sh
npm i @modus/more --registry=https://npm.modus.app/ 
```

This package is in our private registry for now. I'm surprised package-lock didn't pick that up